gemelli/q2/tests/simulations.py

```diff
@@ -0,0 +1,244 @@
+from __future__ import division
+# utils
+import pandas as pd
+import numpy as np
```

Comment:
Can I suggest a different location for this code than a `tests` directory? While used in your testing, I could see this code being pretty useful to others outside of a 'unit testing' context, in which it feels weird to import from `tests`.

Maybe something like `gemelli.simulations`?

Reply:
The simulations have been moved to the main package.